cli/kill.go

```diff
@@ -115,7 +116,20 @@ func kill(containerID, signal string, all bool) error {
 		return nil
 	}
 
-	_, err = vci.StopContainer(sandboxID, containerID)
+	containerType, err := oci.GetContainerType(status.Annotations)
+	if err != nil {
+		return err
+	}
+
+	switch containerType {
+	case vc.PodSandbox:
```

Comment:
@sboeuf when do crio/cri-containerd/docker-shim etc. set the podsandbox annotation? It is set for the first container created in a sandbox?

Reply:
It is set for the container representing the sandbox, and yes it is usually the first one. In case of CRIO, they create what they call an `infra` container as being the sandbox, and then they spawn a bunch of other containers later.